.gitignore

```diff
@@ -7,3 +7,6 @@
 .*.swp
 /.vscode
 
+/.idea
+
+*.iml
```

Comment:
What are these files that are getting ignored?

Reply:
These are file created by the IDE I'm using: IntelliJ

Reply:
Ok, seems reasonable.   Maybe add a comment?